server/kitsu/push.py

```diff
@@ -221,14 +223,15 @@ async def generate_user_settings(
 async def sync_person(
     addon: "KitsuAddon",
     user: "UserEntity",
+    existing_users: dict[str, Any],
     entity_dict: "EntityDict",
 ):
-    logging.info("sync_person")
-
-    username = remove_accents(
-        f"{entity_dict['first_name']}.{entity_dict['last_name']}".lower().strip()
+    logging.info(
```

Comment:
Can we store the first and last name to variables? There is a lot `entity_dict['first_name']` and `entity_dict['last_name']` everywhere.

Reply:
Good idea, addressed in [ef114ae](https://github.com/ynput/ayon-kitsu/pull/55/commits/ef114ae711fe2caecfd50bf07ce911f27e6b963c)